src/components/popups/transferrequestpopup.vue

```diff
@@ -140,6 +181,15 @@
                 </ui-item-text-content>
             </ui-item>
         </ui-list>
+        
+        <ui-alert v-if="warning" state="warning">
+            {{
+                warning === "serverError"
+                    ? "Couldn't check if account is safe, proceed with caution."
```

Comment:
If a user sees this he could think that accounts that do not have the warning are safe, this delivers way too much safety feeling

Reply:
Do you want a default warning of caution? Or do you have a proposal for an alternative string?